index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Aimal Amiri</title>
+
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@700&display=swap" rel="stylesheet" />
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <nav class="container">
+      <a href="#" class="navbar-logo font-inter text-bold decoration-none"> Aimal Amiri </a>
+      <a href="#" class="navbar-menu-button">
+        <img src="./assets/svg/menu.svg" alt="Menu" />
+      </a>
+    </nav>
```

Comment:
# Required Changes
- [ ] You are doing well so far in structuring your navbar. You should have a `<header>` tag inside of which you place the logo in an `<a>` tag as you already have it and then your `<nav>` tag.  Please take the `<a>` tag holding the logo outside of the nav.